ui/app/templates/components/job-page/sysbatch.hbs

```diff
@@ -5,12 +5,16 @@
 
   <JobPage::Parts::StatsBox @job={{this.job}} />
 
-  <JobPage::Parts::JobClientStatusSummary
-    @job={{this.job}}
-    @jobClientStatus={{this.jobClientStatus}}
-    @gotoClients={{this.gotoClients}} />
-
-  <JobPage::Parts::Summary @job={{this.job}} @forceCollapsed="true" />
+  {{#if (can "read client")}}
```

Comment:
This check can go into the `JobClientStatusSummary`.

Reply:
Unfortunately it cant'...as soon as Ember accesses `jobClientStatus` it will make an API request to get the Node information, which will 403.

I moved the check to the param access instead, does this look better?
```hbs
<JobPage::Parts::JobClientStatusSummary
  @job={{this.job}}
  @jobClientStatus={{if (can "read client") this.jobClientStatus}}
  @forceCollapsed={{cannot "read client"}}
  @gotoClients={{this.gotoClients}} />
```

Reply:
Hmmm... I think this breaks my mental model of Ember, and my understanding of the Ember Way.

Currently, we're loading all the nodes into the store when we enter the jobs/job route. And then, we peek the store (no API call) when we're in a nested controller. To my knowledge this isn't a data loading component either.

Can we move this conversation to Whimsical and map this out? That behavior sounds like a lurking bug.

Reply:
> Currently, we're loading all the nodes into the store when we enter the jobs/job route.

That's the problem I'm solving: we don't want to do this if the user's ACL token doesn't have permission to access the `/v1/node`  API endpoint.

But if we try to read a node from the store, Ember will attempt to fetch it but will receive a `403`.

Not sure if this what you're looking for, but this is the flow:

![image](https://user-images.githubusercontent.com/775380/146079823-4f0ec62b-4f64-429f-8de2-e3c053ad7e93.png)

Reply:
1. We're using the computed macro `jobClientStatus` in the wrong place. `JobClientStatusSummary` is the lowest common ancestor that should compute and store this derived state.
2. The API call happens on the route which is protected by the `ember-can` check. The associated controllers aren't fetching `nodes` (with the exception of the `jobs/job/clients` controller, and this tab should be protected using the can helper -- this is another associated bug for this issue) because they're peaking from the store. The component is not fetching data either. Now, there's one more concern, the computed macro `jobClientStatus` this should not be making any API calls, if it is then its a bug.